docs/architecture/incremental-append.md

```diff
@@ -101,7 +101,11 @@ When replicating data incrementally, Airbyte provides an at-least-once delivery
 
 ## Known Limitations
 
-Due to the use of a cursor column, if modifications to the underlying records are made without properly updating the cursor field, then the updated records won't be picked up by the **Incremental - Append** sync as expected.
+Due to the use of a cursor column, if modifications to the underlying records are made without properly updating the cursor field, then the updated records won't be picked up by the **Incremental** sync as expected because the source connectors is extracting delta rows using a SQL query looking like:
```

Comment:
```suggestion
Due to the use of a cursor column, if modifications to the underlying records are made without properly updating the cursor field, then the updated records won't be picked up by the **Incremental** sync as expected since the source connectors extract delta rows using a SQL query looking like:
```